Manager/CommentManager.php

```diff
@@ -72,10 +72,10 @@ public function addComment(
             $thread->setTitle($threadTitle);
         }
 
-        $this->dispatcher->dispatch(Events::PRE_PERSIST_EVENT, new CommentEvent($type, $entityId, $comment, $thread));
+        $this->dispatcher->dispatch(new CommentEvent($type, $entityId, $comment, $thread), Events::PRE_PERSIST_EVENT);
```

Comment:
We should refractor the Events to use for Every Event an own class.
E.g. `CommentPersistEvent`

Like in https://symfony.com/blog/new-in-symfony-4-3-simpler-event-dispatching

Reply:
you are right 🙂 but this does not trigger a deprecation-error right now, so it will probably work for quite some time into the future

Reply:
You are right!